chapters/external-repository-identifiers.md

```diff
@@ -62,6 +64,42 @@ External Reference Site: [https://nvd.nist.gov/cpe](https://nvd.nist.gov/cpe)
 
 Documentation: [http://csrc.nist.gov/publications/nistir/ir7695/NISTIR-7695-CPE-Naming.pdf](http://csrc.nist.gov/publications/nistir/ir7695/NISTIR-7695-CPE-Naming.pdf)
 
+### F.2.3 advisory <a name="advisory"></a>
+
+Locator Format: URL as defined by <https://www.ietf.org/rfc/rfc1738.txt>.
+
+Contextual Example: <https://nvd.nist.gov/vuln/detail/CVE-2020-28498>.
+
```

Comment:
Suggest adding a Contextual Example for an "impact statement" artifact to help people understand the distinction.

Reply:
We working on creating an annex to SPDX specification with examples for various scenarios.

Reply:
Thanks.